pkg/cmd/roachtest/backup.go

```diff
@@ -18,38 +18,56 @@ import (
 
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/cockroach/pkg/util/version"
-	"github.com/pkg/errors"
+	"github.com/cockroachdb/errors"
+)
+
+// The following env variable names match those specified in the TeamCity
+// configuration for the nightly roachtests. Any changes must be made to both
+// references of the name.
+const (
+	KMSRegionAEnvVar = "AWS_KMS_REGION_A"
+	KMSRegionBEnvVar = "AWS_KMS_REGION_B"
+	KMSKeyARNAEnvVar = "AWS_KMS_KEY_ARN_A"
+	KMSKeyARNBEnvVar = "AWS_KMS_KEY_ARN_B"
 )
 
 func registerBackup(r *testRegistry) {
+	importBankData := func(ctx context.Context, rows int, t *test, c *cluster) string {
```

Comment:
This was just a refactor which I had done, there was no logic change - https://github.com/cockroachdb/cockroach/pull/52893